sdl_viewer/src/lib.rs

```diff
@@ -38,4 +40,6 @@ pub mod color;
 
 pub mod graphic;
 
```

Comment:
let's remove all the whitespaces between the `mod` lines.

Reply:
ok